src/entities/AuthorizationToken.ts

```diff
@@ -0,0 +1,33 @@
+import moment from "moment";
+
+export type AuthorizationTokenData = {
+  access_token: string;
+  expires_in: number;
+  token_type: string;
+  scope: string[];
+  refresh_token: string;
+  patient: string;
+  expires_at?: number;
+};
+
+export class AuthorizationToken {
+  public accessToken: string;
+  public expiresIn: number;
+  public expiresAt: number;
+  public tokenType: string;
+  public scope: string[];
+  public refreshToken: string;
+  public patient: string;
+
+  constructor(authToken: AuthorizationTokenData) {
+    this.accessToken = authToken.access_token;
+    this.expiresIn = authToken.expires_in;
+    this.expiresAt = authToken.expires_at
+      ? authToken.expires_at
+      : moment().add(this.expiresIn).valueOf();
```

Comment:
moment().valueOf() return epoch time in milli-seconds, and this.expiresIn is in seconds (36000), probably need conversion.

Reply:
@JFU-GIT Good catch!  TY!   Will update this to be:

```
      : moment().add(this.expiresIn * 1000).valueOf();

```